corehq/apps/enterprise/enterprise.py

```diff
@@ -40,8 +41,15 @@ class EnterpriseReport:
 
     DATE_ROW_FORMAT = '%Y/%m/%d %H:%M:%S'
 
-    title = _('Enterprise Report')
-    subtitle = ''
+    @property
+    @abstractmethod
+    def title(self):
+        pass
+
+    @property
+    @abstractmethod
+    def description(self):
```

Comment:
Something feels off about the enterprise report containing a `description`. I feel like this isn't really a property of the report, but rather the 'view model'. But given how the code is now structured, I'm not sure I'd recommend ripping out this property and creating separate view models. How would you feel about renaming it to like `summary_description`, as it seems to describe the summary information more than the entire report

Reply:
Or maybe `total_description`, given the report has a total property?

Reply:
Sure, I think either of those work, although `total_description` is probably more straight-forward. `description_of_total` sounds a little odd

Reply:
55df4bd451cfa66b148d72b28fda998a978bf589